keras_core/backend/jax/distribution_lib.py

```diff
@@ -0,0 +1,56 @@
+"""!!!DO NOT USE!!!
+
+Distribution related class for JAX backend.
+
+This is just a prototype and we might want to unify it
+with other backends in the future.
+"""
+import jax
+
+
+def list_devices(device_type=None):
+    """Return all the available devices based on the device type.
+
+    Note that this should return the global devices in a distributed setting.
+
+    Args:
+        device_type: string of `"CPU"`, `"GPU"` or `"TPU"`. Default to `GPU` or
+            `TPU` if available when device_type is not provided. Otherwise
+            will return the `CPU` devices.
+
+    Return:
+        List of devices that are available for distribute computation.
+    """
+    device_type = device_type.lower() if device_type else None
```

Comment:
This means that JAX will be resolving its own defaults. This is fine, but we'll need to make sure we get the same default behavior in all backends, when we add TF etc.